hera_qm/utils.py

```diff
@@ -149,6 +149,10 @@ def get_metrics_ArgumentParser(method_name):
         a.add_argument('--waterfalls', default=None, type=str, help='comma separated '
                        'list of npz files containing waterfalls of flags to broadcast '
                        'to full flag array and union with flag array in flag_file.')
+        a.add_argument('--output_npz', default=True, type=bool,
```

Comment:
Is this the "decision waterfall" that we were talking about on the telecon? If it is, then maybe make the docs explicit to reference this.

Reply:
Whoops, I was actually not even storing the waterfall itself. I'll add a better description, and store the waterfall (if applicable).